website/docs/extensions/install/index.md

```diff
@@ -8,24 +8,28 @@ keywords: [podman desktop, extension, publishing]
 
 # Installing a Podman Desktop extension
 
-Consider installing Podman Desktop extensions to enrich the default capabilities of Podman Desktop.
+Installing an extension is a great way to expand the capability of Podman Desktop.
 
 #### Prerequisites
 
-- The extension Open Container Initiative (OCI) image name, such as: `quay.io/fbenoit/my-first-extension`.
-
-  See [Publishing a Podman Desktop extension](/docs/extensions/publish).
+- Extensions are packaged as container images. The extension will either direct you to the catalog tab, or provide you with a link to the image (ex. `quay.io/example/extension-name`).
 
 #### Procedure
 
-1. Go to the **<Icon icon="fa-solid fa-cog" size="lg" /> Settings > Extensions**.
+1. Click on the extensions icon.
```

Comment:
'**Extensions** icon'. Since it is a UI element like the Containers page it should be capitalized.